Sources/AppStoreConnectCLI/Model/Build.swift

```diff
@@ -2,124 +2,125 @@
+    init(_ build: AppStoreConnect_Swift_SDK.Build ,_ includes: [AppStoreConnect_Swift_SDK.BuildRelationship]?) {
 
-    let relationships = build.relationships
+        let relationships = build.relationships
 
-    let includedApps = includes?.compactMap { relationship -> AppStoreConnect_Swift_SDK.App? in
-      if case let .app(app) = relationship {
-        return app
-      }
-      return nil
-    }
+        let includedApps = includes?.compactMap { relationship -> AppStoreConnect_Swift_SDK.App? in
+            if case let .app(app) = relationship {
+                return app
+            }
+            return nil
+        }
 
-    let includedPrereleaseVersions = includes?.compactMap { relationship -> AppStoreConnect_Swift_SDK.PrereleaseVersion? in
-      if case let .preReleaseVersion(prereleaseVersion) = relationship {
-        return prereleaseVersion
-      }
-      return nil
-    }
+        let includedPrereleaseVersions = includes?.compactMap { relationship -> AppStoreConnect_Swift_SDK.PrereleaseVersion? in
+            if case let .preReleaseVersion(prereleaseVersion) = relationship {
+                return prereleaseVersion
+            }
+            return nil
+        }
 
-    let includedBuildBetaDetails = includes?.compactMap { relationship -> AppStoreConnect_Swift_SDK.BuildBetaDetail? in
-      if case let .buildBetaDetail(buildBetaDetail) = relationship {
-        return buildBetaDetail
-      }
-      return nil
-    }
+        let includedBuildBetaDetails = includes?.compactMap { relationship -> AppStoreConnect_Swift_SDK.BuildBetaDetail? in
+            if case let .buildBetaDetail(buildBetaDetail) = relationship {
+                return buildBetaDetail
+            }
+            return nil
+        }
 
-    let includedBetaAppReviewSubmissions = includes?.compactMap { relationship -> AppStoreConnect_Swift_SDK.BetaAppReviewSubmission? in
-      if case let .betaAppReviewSubmission(betaAppReviewSubmission) = relationship {
-        return betaAppReviewSubmission
-      }
-      return nil
-    }
+        let includedBetaAppReviewSubmissions = includes?.compactMap { relationship -> AppStoreConnect_Swift_SDK.BetaAppReviewSubmission? in
+            if case let .betaAppReviewSubmission(betaAppReviewSubmission) = relationship {
+                return betaAppReviewSubmission
+            }
+            return nil
+        }
 
-    let appDetails = includedApps?.filter { relationships?.app?.data?.id == $0.id }.first
-    let prereleaseVersion = includedPrereleaseVersions?.filter { relationships?.preReleaseVersion?.data?.id == $0.id }.first
-    let buildBetaDetail = includedBuildBetaDetails?.filter { relationships?.buildBetaDetail?.data?.id == $0.id }.first
-    let betaAppReviewSubmission = includedBetaAppReviewSubmissions?.filter { relationships?.betaAppReviewSubmission?.data?.id == $0.id }.first
+        let appDetails = includedApps?.filter { relationships?.app?.data?.id == $0.id }.first
```

Comment:
Shouldn't we use first(where: ) here as we discussed yesterday for listprereleaseversion command?

Reply:
Potentially, however this was just a straight format of the file. I might raise a separate PR for it since it's not the focus of this PR